lib/mix/tasks/monarch.ex

```diff
@@ -16,24 +16,24 @@ defmodule Mix.Tasks.Monarch do
   def run(args) do
     case OptionParser.parse!(args, switches: @switches) do
       {opts, [base_name]} ->
-        path = opts[:monarch_path] || :code.priv_dir(:my_app)
-
-        app_dir = File.cwd!()
-        file_name = "#{base_name}.ex"
-        file = Path.join([app_dir, path, file_name])
-        unless File.dir?(path), do: create_directory(path)
+        path = opts[:monarch_path]
```

Comment:
🤔 Looks like my issue was trying to call `:code.priv_dir(:my_app)` which doesn't work at compile time.